src/components/Title.tsx

```diff
@@ -2,16 +2,41 @@ import React from 'react';
 import styled from 'styled-components';
 import { LeftDirection, Close } from '@/assets/icons';
 
-const Title = () => {
+interface TitleProps {
+  title: string;
+  icon: 'close' | 'arrow';
+}
+
+const Title = ({ title, icon }: TitleProps) => {
   return (
-    <div>
-      title wrapper
-      <Close />
-      <div>Title</div>
-    </div>
+    <Wrapper iconType={icon}>
+      <div>{icon === 'close' ? <Close /> : <LeftDirection />}</div>
+      <h2>{title}</h2>
+    </Wrapper>
   );
 };
 
 export default Title;
 
-const TitleWrapper = styled.div``;
+const Wrapper = styled.div<{ iconType: string }>`
+  width: 100%;
+  padding: 10px;
+  margin-bottom: 10px;
+  border-bottom: 1px solid ${({ theme }) => theme.color.border.lightgray};
+  position: relative;
+  ${({ theme, iconType }) =>
+    iconType === 'close'
+      ? theme.mixins.flexBox('center', 'center')
+      : theme.mixins.flexBox('center', 'flex-start')};
+  div {
+    position: absolute;
+    left: 10px;
+    top: 50%;
+    transform: translateY(-50%);
+  }
+  h2 {
+    font-size: 1.3rem;
+    line-height: 2rem;
+    margin: 0 50px;
+  }
+`;
```

Comment:
position: relative, absolute가 필요한 지 모르겠어서 수정해보았습니다 참고 부탁드려요!
```suggestion
const Wrapper = styled.div<{ iconType: string }>`
  width: 100%;
  padding: 15px 5px;
  margin-bottom: 10px;
  border-bottom: 1px solid ${({ theme }) => theme.color.border.lightgray};
  ${({ theme }) => theme.mixins.flexBox('center', 'center')};

  div {
    flex: ${({ iconType }) => (iconType === 'close' ? 0 : 1)};
    padding-right: ${({ iconType }) => (iconType === 'close' ? '20px' : 0)};
    text-align: left;
    box-sizing: border-box;
  }
  h2 {
    flex: 2;
    font-size: 1.3rem;
  }
`;
```

Reply:
flex를 이용한 자리배치 너무 좋은 의견입니다. 잘 반영했습니다!!